packages/cli/src/lib/tasks.ts

```diff
@@ -69,6 +69,7 @@ export async function templatingTask(
   templateDir: string,
   destinationDir: string,
   context: any,
+  versions: { [name: string]: string },
```

Comment:
`packageVersions: { [packageName` to be extra clear?

Reply:
Yeah `packageVersions` might be a bit more clear